login.js

```diff
@@ -0,0 +1,81 @@
+const emailInput = document.getElementById("userName");
+const passwordInput = document.getElementById("userPassword");
+
+const errorMessage = document.getElementById("error-message");
+const errorPassword = document.getElementById("error-password");
+
+const loginBtn = document.getElementById("loginBtn");
+
+document.addEventListener("DOMContentLoaded", () => {
+  emailInput.addEventListener("input", () => {
+    if (emailInput.value !== "" && emailInput.value.includes("@")) {
```

Comment:
💊 제안
바로 email input의 값이 비어있는지 확인하게 되면 "  " 와 같이 공백만 존재하는 경우에도 값이 있다고 판단하게 됩니다.
따라서 string의 trim method를 통해 공백을 제거하시면 의도한 바와 더 가까울 것 같습니다.
조건문에서 이메일 형식을 확인하는 `emailInput.value.includes("@")` 부분의 경우도 @ 라는 요소가 들어가있는지만 확인하는 것이라 명확한 이메일 형식 확인은 되지 않고 있으니 이메일 관련 정규식을 통해 구현하시면 더 좋겠습니다~

```suggestion
  emailInput.addEventListener("input", () => {
    if (emailInput.value.trim() !== "" && emailInput.value.includes("@")) {
```